appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -368,32 +363,22 @@ class LoggedInFlowNode @AssistedInject constructor(
         }
     }
 
-    suspend fun attachRoomList() {
-        if (!canShowRoomList()) return
-        attachChild<Node> {
-            backstack.singleTop(NavTarget.RoomList)
-        }
-    }
-
     suspend fun attachRoom(roomId: RoomId) {
-        if (!canShowRoomList()) return
+        waitForNavTargetAttached { navTarget ->
```

Comment:
mmh, I did not test the use case of tapping a notification (neither tapping on multiple ones) when a ftue screen is displayed, I will test it.

But if at the end the room is displayed, I think this is the expected behavior.

As I said in the description, if the user clicks on a room permalink from outside the app, the user may have to log in first and at the end will se the room, because this method is suspending.

Reply:
IMO that behaviour is bit weird, what if: 

1. The user taps on a notification for a room, it opens the app with FTUE displayed.
2. Then they tap on a notification for another room.
3. They finish the FTUE flow.

What will happen then? Will the 1st room be opened, the 2nd one, or both one after the other?

Reply:
Yes, I just checked and all the rooms will be added to the backstack, and several back press will be necessary to go back to the room list. I think this is OK since:
- it will not happen a lot (FTUE screen are only for the first time, by definition)
- the user has performed action (click) so is expecting to see the room at some point.

We could ensure that the same room is not added twice on the backstack (i.e. click on a first notification, then get another one for the same room and click on this new one), but really this is edge cases, and this can be handled later if this is a reported issue.

Reply:
I think we should handle it here since we have the chance now, but if it's blocking some other PR I can approve it as is and we can handle it later.